lib/dependencies/HarmonyExportExpressionDependency.js

```diff
@@ -26,7 +56,15 @@ class HarmonyExportExpressionDependency extends NullDependency {
 	}
 }
 
+/**
+ * @implements {DependencyTemplate}
```

Comment:
Unfortunately, `@implements` does not work for Typescript yet, and workaround with `{{prototype: DependencyTemplate}}` does not as well. Workaround works only for definition (object still is not callable, fails on `new Template()`). So I implemented each method to support Typescript in templates.

Reply:
You could create a `DependencyTemplate` class and `extends DependencyTemplate` for all dep templates

Reply:
done